CONTRIBUTING.md

```diff
@@ -146,6 +146,10 @@ If you only want subset of these actions, you can use:
 
 - `npm run watch:build` to start a watcher task that **re-compiles Vega-Lite** when `.ts` files related to VL change.
 
+#### Fast iteration testing
```

Comment:
Fast Unit Testing

What you suggests below only runs unit test, but not example validation test. 
("other things required for testing" is vague.)  

Since this won't make test really pass on Travis, I wonder if we should really include this in CONTRIBUTING.md.